layers/+tools/systemd/README.org

```diff
@@ -25,7 +29,18 @@ To use this contribution add it to your =~/.spacemacs=
 
 * Key bindings
 
-| Key binding | Description                                                              |
-|-------------+--------------------------------------------------------------------------|
-| ~SPC m h d~ | Systemd open the directives documentation                                |
-| ~SPC m h o~ | Systemd open documentation (if a =\=Documentation= directive is provided |
+| Key binding   | Description                                                              |
+|---------------+--------------------------------------------------------------------------|
+| ~SPC m h d~   | Systemd open the directives documentation                                |
+| ~SPC m h o~   | Systemd open documentation (if a =\=Documentation= directive is provided |
+
+* Journalctl-mode
+This is a major-mode for emacs to view systemd's journalctl output in emacs. The output is split into chunks for performance reasons. Fontification is provided and may be customized.
+
+** Key Bindings
```

Comment:
the key bindings for journalctl-mode should be added under the `* Key bindings` section